etc/kayobe/kolla/config/keystone/wsgi-keystone.conf

```diff
@@ -0,0 +1,63 @@
+{% raw %}
+{% set keystone_log_dir = '/var/log/kolla/keystone' %}
+{% set python_path = '/usr/lib/python2.7/site-packages' if kolla_install_type == 'binary' else '/var/lib/kolla/venv/lib/python2.7/site-packages' %}
+Listen {{ api_interface_address }}:{{ keystone_public_port }}
+Listen {{ api_interface_address }}:{{ keystone_admin_port }}
+
+TraceEnable off
```

Comment:
Perhaps worth having a comment separating the change you've made from the base wsgi conf file?

Reply:
Inline you mean, or as part of the commit message?

Reply:
I was just thinking an inline comment marking your customisation from the base config, or perhaps two commits, one of the base config file, and the other with the customisation for ```git blame```. Not to worry too much.

Reply:
I thought about two discrete commit but then there's still no way to guarantee provenance.  Point taken about traceability with `git blame` though, at least in order to narrow scope.